rendering/glb.ksy

```diff
@@ -0,0 +1,69 @@
+meta:
+  id: glb
+  title: GL Transmission Format, binary container
+  file-extension: glb
+  endian: le
+  license: MIT
+  xref:
+    mime: model/gltf-binary
+    wikidata: Q28135989
+
+doc: |
+  glTF is a format for distribution of 3D models optimized for being used in software
+
+doc-ref: https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#binary-gltf-layout
+
+seq:
+  - id: header
+    type: header
+  - id: chunks
+    type: chunk
+    repeat: eos
+
+types:
+
+  header:
+    seq:
+      - id: magic
+        contents: glTF
+      - id: version
+        type: u4
+      - id: length
+        type: u4
+
+  chunk:
+    seq:
+      - id: length
+        type: u4
+      - id: type
+        type: u4
+        enum: chunk_type
+      - id: data
+        size: length
+        type:
+          switch-on: type
+          cases:
+            'chunk_type::json': json
+            'chunk_type::bin': bin
+
+  json:
+    seq:
+      - id: data
+        type: str
+        encoding: UTF-8
+        size-eos: true
```

Comment:
The order à la [style guide](https://doc.kaitai.io/ksy_style_guide.html#seq-attr):

```suggestion
      - id: data
        size-eos: true
        type: str
        encoding: UTF-8
```

There is a logic behind it: when this field is parsed, first an intermediate byte array till the `e`nd `o`f `s`tream is read, which is then interpreted as a `str`ing in the specified `encoding`.